### ISSUE TITLE
Documentation: insert a one-line explanation about `noop` on manual

### ISSUE THREAD

**Issue by pdelfino** [atlas-engineer/nyxt#1658]
On the subsection *keybinding configuration* of manual, there should be a brief mention to `noop`.

A few weeks ago, when I was tweaking with config files I was craving for that, I just did not know it already existed and the thing had this name. Today, accidentally, I uncover this command `noop`:

> A command that does nothing. This is useful to override bindings to do nothing.

At least, it could be used as **part of the example already available**: 

[...] 
The override-map is a keymap _that_ has priority over all other keymaps. By default, it has few bindings like the one for execute-command. You can use it to set keys globally (last line of the expression):

```
(define-configuration buffer
  ((override-map (let ((map (make-keymap "override-map")))
                   (define-key map
                     "M-x" 'execute-command
                     "C-space" 'noop)))))
```

Beyond the insertion on the example, there could be a brief explanation/mention right below the code:
 
[...] _The_ `noop` _command is useful to override bindings to nothing_. _In addition_, a more flexible _approach_ is to create [...]

I would be happy to send a patch.

**Comment by Ambrevar** [comment]
Sure thing, feel free to send a pull request.

If it makes it easier for you, especially for documentation, it's often
faster to send a pull request directly than opening an issue about it ;)

**Comment by pdelfino** [comment]
> Sure thing, feel free to send a pull request. If it makes it easier for you, especially for documentation, it's often faster to send a pull request directly than opening an issue about it ;)

Wow, I am glad to hear this!
I was reading about open source projects and the recommendation was to open issues before sending a PR. But I was feeling this was somewhat too much bureaucratic, at least for minor changes or documentation.

I think just sending the PR will make me more productive. Thanks for the hint.

**Comment by pdelfino** [comment]
I am closing this issue since the suggestions have now been merged with  PR #1672.
